shims/spark311/src/main/scala/org/apache/spark/sql/execution/datasources/parquet/rapids/shims/spark311/ParquetMaterializer.scala

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2020-2021, NVIDIA CORPORATION.
+ * Copyright (c) 2021, NVIDIA CORPORATION.
```

Comment:
put 2020 back

Reply:
that is an unfortunate side effect of deferring full PCBS consolidation to another PR